qutebrowser/config/configdata.yml

```
@@ -371,11 +371,25 @@ content.default_encoding:
     _iso-8859-1_, etc.
 
 content.windowed_fullscreen:
+  renamed: content.fullscreen.window
+
+content.fullscreen.window:
   type: Bool
   default: false
   desc: >-
     Limit fullscreen to the browser window (does not expand to fill the screen).
 
+content.fullscreen.overlay_timeout:
+  type: 
+    name: Int
+    minval: 0
+    maxval: maxint
+  default: 3000
+  desc: >-
+    Set fullscreen notification overlay timeout in miliseconds.
+    
+    Set 0 for not displaying it at all
```

For consistency with other descriptions:

```suggestion
    If set to 0, no overlay will be displayed.
```